### ISSUE TITLE
maven fails to resolve 0.1.10

### ISSUE THREAD

**Issue by jottinger** [gbenroscience/ParserNG#35]
pom.xml: 
```xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>com.enigmastation</groupId>
    <artifactId>demo1</artifactId>
    <version>1.0-SNAPSHOT</version>

    <properties>
        <maven.compiler.source>21</maven.compiler.source>
        <maven.compiler.target>21</maven.compiler.target>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    </properties>

    <dependencies>
        <dependency>
            <groupId>com.github.gbenroscience</groupId>
            <artifactId>parser-ng</artifactId>
            <version>0.1.10</version>
        </dependency>
    </dependencies>
</project>
```

No source or anything: the error is demonstrable with this.

`mvn package -U` yields this:

```
[INFO] Scanning for projects...
[INFO]
[INFO] ----------------------< com.enigmastation:demo1 >-----------------------
[INFO] Building demo1 1.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
Downloading from central: https://repo.maven.apache.org/maven2/com/github/gbenroscience/parser-ng/0.1.10/parser-ng-0.1.10.pom
[WARNING] The POM for com.github.gbenroscience:parser-ng:jar:0.1.10 is missing, no dependency information available
Downloading from central: https://repo.maven.apache.org/maven2/com/github/gbenroscience/parser-ng/0.1.10/parser-ng-0.1.10.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.493 s
[INFO] Finished at: 2024-08-15T14:33:42-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project demo1: Could not resolve dependencies for project com.enigmastation:demo1:jar:1.0-SNAPSHOT
[ERROR] dependency: com.github.gbenroscience:parser-ng:jar:0.1.10 (compile)
[ERROR] 	Could not find artifact com.github.gbenroscience:parser-ng:jar:0.1.10 in central (https://repo.maven.apache.org/maven2)
[ERROR]
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

**Comment by gbenroscience** [comment]
Thanks for using `ParserNG`.
Please the latest build on maven is **0.1.9**, not 0.1.10

**Comment by gbenroscience** [comment]
So, I went back to oss.sonatype to check the artifacts for version **0.1.10** and I found that the jar was not available. We had this issue with sonatype for version **0.1.9** also, which is why we had to create version **0.1.9-release**
As a workaround, please use the version 0.1.9-release in your project, as there is very little difference between them.
Thanks for spotting this issue.

**Comment by gbenroscience** [comment]
I will close this issue, but feel free to reopen it if you have any other... well, issue with this decision.

**Comment by judovana** [comment]
> So, I went back to oss.sonatype to check the artifacts for version **0.1.10** and I found that the jar was not available. We had this issue with sonatype for version **0.1.9** also, which is why we had to create version **0.1.9-release** As a workaround, please use the version 0.1.9-release in your project, as there is very little difference between them. Thanks for spotting this issue.

Thanx a lot for taking care!
Were you able to resolve the missing jar? Can you plese draft the release on gihub releasese? No build or binaries needed (only that is usually very convenient:) ), but it will show the diff beteen 0.9 and 0.1.10 

And this remained me that I still have lying around that  Double/Bigdecimal patch....

**Comment by jottinger** [comment]
0.1.9-release is fine and working for me. 0.1.10 is still not on maven central in usable form, although the javadoc's uploaded; it's the actual jar that isn't.

**Comment by gbenroscience** [comment]
> > So, I went back to oss.sonatype to check the artifacts for version **0.1.10** and I found that the jar was not available. We had this issue with sonatype for version **0.1.9** also, which is why we had to create version **0.1.9-release** As a workaround, please use the version 0.1.9-release in your project, as there is very little difference between them. Thanks for spotting this issue.
> 
> Thanx a lot for taking care! Were you able to resolve the missing jar? Can you plese draft the release on gihub releasese? No build or binaries needed (only that is usually very convenient:) ), but it will show the diff beteen 0.9 and 0.1.10
> 
> And this remained me that I still have lying around that Double/Bigdecimal patch....

Hi @judovana. Long time. Be content a little bit for now(lol!) I will find time to work on it(Double/BigDecimal patch), please.

I doubt that we can rescue the 0.1.10 thing, except if maybe I delete the repo on maven central and re-push. Will look into these things soon, but since the difference between 0.1.10 and 0.1.9 is basically in description and not in code, I doubt that will be priority for now.
Thanks for showing up again,

**Comment by judovana** [comment]
I have done it that time for that: https://github.com/judovana/CustomJmhBenchmarks/blob/numbers/README.md#parserng-rewritten-from-double-to-double ; it is "just" https://github.com/judovana/ParserNG/commit/352745f80d47333af28c5fce60a766c5c3d63b0b  which should be the first step, and then all can just work with Number, and precissin x fast can be easily selected on startup. Currenlty I can imageind only Double and BigDecimal  (however the https://github.com/judovana/ParserNG/commits/origin/add-bigdecimal-capability/ is not finished). Do you want me to do the double->Double pr? The commit exists, and is tested, it just need a second eyeball and a bit of cleanup. And performance cost is absolute nothing. JVM is doing its job well.


As for 1.10, what I had mind, is really just github diff, easily click-able in release (you can diff tags,but 0.1.10 tag is missing). So maybe just tag? If you know where...  I think that the best to fix the release is to do 0.1.11; that would do.

**Comment by gbenroscience** [comment]
@judovana , thanks a lot. Please send in the PR; but as I usually say, you would have to wait a bit for me so I can review it when I have the time. Thanks for all you do
